### ISSUE TITLE
can't figure out styling template-entitiy-row inside conditional inside picture-elements card

### ISSUE THREAD

**Issue by ktownsend-personal** [thomasloven/lovelace-card-mod#69]
I'm hoping you can assist me with a challenge I am having trying to use `card-mod` to style another of your elements: `template-entity-row`, wrapped by a `conditional` in a `picture-elements` card. 

I need to adjust the style of the name area on `template-entity-row` to remove the left margin and enable flex-shrink because I am using it outside of an entities card and the default styling needed for a row just doesn't look right when it's not used as a row. I already know I can hack the .js file to add an inline style to that div, but that's fragile if I want to keep getting updates via HACS, not to mention it affects other places where I might want to use it as an actual row. 

I found your `card-mod` element, but can't get that to work. From what I can tell in your documentation it's because of the `shadow-root` inside `template-entity-row`, and further complicated by the `conditional` element I'm wrapping around `template-entity-row`. I tried using `custom:mod-card` as your documentation suggests for this case, but I just can't get it to work. 

Below is a trimmed version of my yaml for the `picture-elements` card with just the conditional wrapper around a `template-entity-row`. After that is the style I tried to use. I tried it at the root of the `picture-elements` card, and I tried it with a `custom:mod-card` wrapping the `picture-elements` card.

Do you have any suggestions? I thought about trying to place the `card-mod` style definition directly on the `template-entity-row`, but it already has a dictionary based style config used by the `picture-elements` card and the 2 types of styling syntax are not compatible. 

```yaml
elements:
  - conditions:
      - entity: switch.sprinkler_zone_12
        state: 'on'
    elements:
      - active: '{{ is_state(''switch.sprinkler_zone_12'', ''on'') }}'
        entity: switch.sprinkler_zone_12
        icon: 'hass:cancel'
        name: filling
        state: >
          {% set s = (state_attr('switch.sprinkler_zone_12', 'time_remaining')|int) %}
          {% set h = (s / 3600) %}
          {% set m = (s % 3600) / 60 %} 
          {{ "%02d:%02d" % (h,m) }}
        style:
          background-color: 'rgba(0,0,0,0.4)'
          border-radius: 15px 0 0 0
          bottom: 66px
          color: white
          font-size: 1.2em
          padding-right: 5px
          right: 0
          transform: none
        tap_action:
          action: toggle
        type: 'custom:template-entity-row'
    type: conditional
image: /local/pictures/pool.jpg
type: picture-elements
```

```yaml
style: |
  "template-entity-row":
    $: |
      .info {
        margin-left: 0;
        margin-right: 5px;
        flex-shrink: 1
      }
```

Here is a pic of the DOM, with the target div selected:
(ignore the inline style; that was a direct hack of the js file to achieve the styling I need, which I'd rather not do)
![image](https://user-images.githubusercontent.com/22361432/90214393-ef34e500-ddbd-11ea-99d9-a3322427fb43.png)

**Comment by thomasloven** [comment]
Styling elements in picture-elements cards has not been supported until now in version 3.0.
Please try that.

**Comment by thomasloven** [comment]
So...?

**Comment by ktownsend-personal** [comment]
I haven't had a chance to try it out, but definitely appreciative of the new capability and looking forward to trying it out next time I have a similar situation. I resolved the original issue a while ago by using the custom button-card, which gives me excellent control of layout, styling and compositing multiple entities into one unit. My automation devices and UI are stable, so I've been enjoying not having to tinker with it the past few months, and as a result I have been able to redirect my efforts to other big projects unrelated to home automation.  I have a pile of ideas related to home automation that I will get back to eventually, so I'm sure this need will arise again and I will enjoy the new capability you added.
